expressEngine/lib/service/db.js

```diff
@@ -14,6 +16,11 @@ class DbOperation {
   findOne = (collection, data = {}, projection = {}) => {
     return collection.findOne(data, projection);
   }
+  findAll = ({collection, data = {}, skip = 0, limit = 10 , projection = {}}) => {
+    const skipValue = baseService.parseNumber(skip , 0);
+    const limitValue = baseService.parseNumber(limit, 10);
+    return collection.find(data, projection).skip(skipValue).limit(limitValue);
+  }
 }
 
-export default new DbOperation();
\ No newline at end of file
+export default new DbOperation();
```

Comment:
Replace `DbOperation();` to `DBOperation();`
Reason is mentioned [here](https://github.com/nodexpertsdev/RnD/blob/develop/CODING_GUIDE.md#naming--Acronyms-and-Initialisms)

Reply:
Done sir